frontend/src/styles/modules/_buttons.scss

```diff
@@ -145,6 +145,10 @@
     background-color: $disabled-grey;
   }
 
+  &.big-border {
```

Comment:
I'd kinda prefer to see this as inline CSS...seems like the fewer files touched for A/B tests the better. Alternately, we could have a standalone sass page for variants that are namespaced as such like `class=variant--big-border`